### PR TITLE
Update GitHub Onboarding

### LINKED ISSUE #3935: Update onboarding
### New Feature Description

New flow for onboarding to focus instead on Github integration

### Required Reviewers

ryan, paola

### Links to Internal Info or Resources

https://www.notion.so/astronomerio/Docs-Priorities-2024Q2-e1ca7b96fbdf476ba2450c1b4dfedb79?pvs=4

### Release Date

_No response_

### Additional Notes

_No response_

### PR DESCRIPTION
**Pull request by lzdanski**
Resolves #3935

### CODE REVIEW

File: astro/first-dag-onboarding.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Step 2: Create your Oganization and Workspace
+
+Enter a name for your Organization and Workspace. These can be changed later in the Astro UI.
+
+An Astro _Organization_ is the highest level entity in Astro and represents a shared space for your company on Astro. An Organization is automatically created when you first sign up for Astronomer. At the Organization level, you can manage all of your users, Deployments, Workspaces, and clusters from a single place in the Astro UI. A _Workspace_ is a collection of Deployments that can be accessed by a specific group of users.
+
+## Step 3: Select a template
+
+You can choose a template Astro project that shows a demonstration of particular scenarios. An _Astro project_ contains the set of files necessary to run Airflow, including dedicated folders for your DAG files, plugins, and dependencies.
```
Comment (iancmoritz) [inline astro/first-dag-onboarding.md:59]:
```suggestion
An _Astro project_ contains the set of files necessary to run Airflow, including dedicated folders for your DAG files, plugins, and dependencies. Select a template Astro project to immediately get a demonstration of Airflow's orchestration capabilities.
```

Comment (iancmoritz) [reply astro/first-dag-onboarding.md:59]: small NIT, and I think we could potentially remove or alter line 61 if we go with something like this?

**Review by iancmoritz — APPROVED** [review]
Overall, this looks great and haven't found anything I'd really change here. The only thing I'd consider is having a single image of a completed Step 4 to show what the terminal state of selecting a deployment, connecting Github, etc. is. Thoughts @lzdanski ?

File: astro/first-dag-onboarding.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+These templates include a fully functional Astro project for specific use cases, or just a general project for you to use to explore Airflow.
```
Comment (lzdanski) [inline astro/first-dag-onboarding.md:61]:
```suggestion
```

File: astro/first-dag-onboarding.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+This tutorial takes about 15 minutes. If you're new to Airflow and want a more in-depth tutorial, see [Airflow 101 Learning Path](https://academy.astronomer.io/path/airflow-101).
+
+## Prerequisites
+
+- An Astro account. To start an Astro trial and create your free trial account, see [Start a trial](trial.md).
+- (Recommended) A [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github).
+
+:::info
+
+If you're on your organization's network and can't access Astro, make a request to allowlist the following domains on your network:
```
Comment (merobi-hub) [inline astro/first-dag-onboarding.md:26]:
```suggestion
If you're on your organization's network and can't access Astro, make a request to `allowlist` the following domains on your network:
```

Comment (merobi-hub) [reply astro/first-dag-onboarding.md:26]: maybe this should be formatted as code?

Comment (lzdanski) [reply astro/first-dag-onboarding.md:26]: I doublechecked the google style guide, which we fall back to for word use, and it says to use it as a noun vs a verb, but no code formatting! made this change here, will copy over to other.

File: astro/first-dag-onboarding.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+---
+
+Astro is the industry's leading managed service for [Apache Airflow®](https://airflow.apache.org/). To quickly learn how Astro works, follow the steps in this quickstart to create an Airflow environment and run your first DAG.
+
+When you first sign up for Astro, you can choose to tailor your experience based on how you plan to use Astro. The onboarding process includes:
+
+- Creating an Organization, Workspace, and Deployment
+- Choosing an example Airflow project template to clone into your own sample GitHub Repository
+- Connecting your Astro Deployment to your sample GitHub Repository, so you can deploy code from GitHub
+- Run your first DAG on Astro
```
Comment (merobi-hub) [inline astro/first-dag-onboarding.md:15]:
```suggestion
- Running your first DAG on Astro
```

File: astro/first-dag-onboarding.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+Enter a name for your Organization and Workspace. These can be changed later in the Astro UI.
+
+An Astro _Organization_ is the highest level entity in Astro and represents a shared space for your company on Astro. An Organization is automatically created when you first sign up for Astronomer. At the Organization level, you can manage all of your users, Deployments, Workspaces, and clusters from a single place in the Astro UI. A _Workspace_ is a collection of Deployments that can be accessed by a specific group of users.
+
+## Step 3: Select a template
+
+An _Astro project_ contains the set of files necessary to run Airflow, including dedicated folders for your DAG files, plugins, and dependencies. Select a template Astro project to immediately get a demonstration of Airflow's orchestration capabilities.
+
+
+- **Generative AI**: Airflow is a common orchestration engines for AI/Machine Learning jobs, especially for retrieval-augmented generation (RAG).This [generative AI project](https://github.com/astronomer/templates/blob/main/generative-ai/README.md) shows an simple example of building vector embeddings for text and then performing a semantic search on the embeddings.
```
Comment (merobi-hub) [inline astro/first-dag-onboarding.md:62]:
```suggestion
- **Generative AI**: Airflow is a common orchestration engine for AI/Machine Learning jobs, especially for retrieval-augmented generation (RAG).This [generative AI project](https://github.com/astronomer/templates/blob/main/generative-ai/README.md) shows an simple example of building vector embeddings for text and then performing a semantic search on the embeddings.
```

Comment (merobi-hub) [inline astro/first-dag-onboarding.md:62]:
```suggestion
- **Generative AI**: Airflow is a common orchestration engines for AI/Machine Learning jobs, especially for retrieval-augmented generation (RAG). This [generative AI project](https://github.com/astronomer/templates/blob/main/generative-ai/README.md) shows a simple example of building vector embeddings for text and then performing a semantic search on the embeddings.
```

File: astro/first-dag-onboarding.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+- **Learning Airflow**: This example project is generated when you run 'astro dev init' using the Astro CLI. It shows a basic Astro project with Airflow components and DAGs.
+- **None**: This option allows you to manually deploy a project after making a Deployment. See [Deploy code with GitHub](deploy-github-integration.md) to use a GitHub connection with an existing project in a repo. Or, you can choose to manually build and Deploy an example DAG by following the steps in [Run your first DAG with the Astro CLI](first-dag-cli.md) or [Run your first DAG with GitHub Actions](first-dag-github-actions.md).
+
+## Step 4: Set up your Astro Deployment
+
+You can now confirm your selected template and finish configuring your Deployment resources and whether you want to use a GitHub connection for code deploys.
+
+An Astro _Deployment_ is an instance of Apache Airflow that is powered by all core Airflow components, including a webserver, scheduler, and one or more workers. You deploy DAGs to a Deployment, and you can have one or more Deployments within a Workspace.
```
Comment (merobi-hub) [inline astro/first-dag-onboarding.md:71]:
```suggestion
An Astro _Deployment_ is an instance of Apache Airflow that is powered entirely by core Airflow components, including a webserver, scheduler, and one or more workers. You deploy DAGs to a Deployment, and you can have one or more Deployments within a Workspace.
```

File: astro/first-dag-onboarding.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+- **Selected template**: When you finalize your Deployment, you can confirm or change the template option you selected in Step 3.
+- **Git connection**: Enter your **GitHub Owner** and **Repository Owner** information to deploy code directly to an Astro Deployment when you merge changes to a specific branch. If you choose to skip connecting to GitHub, later, you can return and manually build and Deploy an example DAG by following the steps in [Run your first DAG with the Astro CLI](first-dag-cli.md) or [Run your first DAG with GitHub Actions](first-dag-github-actions.md).
+- **Review Deployment**: Enter a **Name** for your Deployment and choose the **Provider** that you want to use and **Region** where you'd like to host your resources.
+
+After you finish configuring your Deployment, click **Create your Project Repository and Deploy to Astro**.
+
+## Step 5: Run your first DAG on Astro
+
+When you create your project and deploy it to Astro, it automatically creates a new Deployment and makes a private GitHub repository in your account with the template that you selected. Then, you can open Airflow and run your DAG.
```
Comment (merobi-hub) [inline astro/first-dag-onboarding.md:81]:
```suggestion
When you create your project and deploy it to Astro, this automatically creates a new Deployment and makes a private GitHub repository in your account with the template that you selected. Then, you can open Airflow and run your DAG.
```

**Review by merobi-hub — APPROVED** [review]

**Pull request merged**